tests/test_events.py

```diff
@@ -18,6 +18,40 @@ def test_basic(listener, sender):
     assert len(listener.received_events) == 3
     assert listener.received_events == names
 
+def test_basic_1_1(listener, sender):
```

Comment:
Since eaf4c8102, there's no added code to test.  I think this test can be removed